keras_cv/layers/object_detection/roi_generator.py

```diff
@@ -23,6 +23,7 @@
 from keras_cv import bounding_box
 from keras_cv.api_export import keras_cv_export
 from keras_cv.backend import assert_tf_keras
+from keras_cv.backend import ops
 
```

Comment:
this file is still not converted to Keras 3 and it does error out with faster RCNN test.

Reply:
please use this - https://github.com/keras-team/keras-cv/blob/7ee18a1efe963b9c5e532c87ae88fbe01a621f7c/keras_cv/layers/object_detection/non_max_suppression.py

Reply:
@divyashreepathihalli I have made the changes as asked. Could you review and see if the changes make sense to you?

Reply:
LGTM!